runtime/src/accounts_db.rs

```diff
@@ -1592,9 +1592,16 @@ impl AccountsDB {
         }
 
         scan.stop();
-        debug!("{}", scan);
 
-        Ok(Self::accumulate_account_hashes(hashes))
+        let mut accumulate = Measure::start("accumulate");
+        let accumulated_hash = Self::accumulate_account_hashes(hashes);
+        accumulate.stop();
+        datapoint_info!(
+            "update_accounts_hash",
+            ("accounts_scan", scan.as_us(), i64),
+            ("hash_accumulate", accumulate.as_us(), i64),
```

Comment:
Do we also want to add a metric for number of accounts/hashes acumulated?

Reply:
added